### PR TITLE
#171648107 Setting up starter project

### PR DESCRIPTION
**Pull request by Irutingabo**
#### Description
This chore adds:
- A react project created from scratch using Webpack 4, Babel, project structure, and the Airbnb linting option.
- A `README` file includes; 
- [X] Installation and run commands
- [X] Description
- [X] Dependencies
- [X] Tests setup and execution

#### Type of change

Please select the relevant option

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Others (cosmetics, styling, improvements)
- [ ] This change requires a documentation update

#### How Has This Been Tested?

- [ ] Unit
- [ ] Integration
- [ ] End-to-end
-  [X]  No testing required

#### How to Test 
Setup:

 1. Clone this repo on your machine
 2. Run `npm i ` to install dependencies
 5. Do `npm run dev` command

Test:

 1. Open your preferred browser, such google chrome browser 
 2. Enter this link in the URL `localhost:8080,
 3. Then hit the send button.

Screenshots or GIFs
**Response**:
image

#### STORY ID
[Starts #171648107]

### CODE REVIEW

File: routes.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "/api/v1/*": "/$1"
```
Comment (placideirandora) [inline routes.json]: What's the purpose of this?

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+module.exports = {
+    entry: "./public/index.js",
+    output: {
+        path: path.join(__dirname, 'build'),
+        filename: 'index.bundle.js'
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
```
Comment (placideirandora) [inline webpack.config.js]: Add  **JSX** file extension support as well because We'll be giving **React Components** the **JSX** extension.

**Review by placideirandora — CHANGES_REQUESTED** [review]
What's the purpose of those **.empty** files inside the **source** folder.
I will propose an Improvement of the Folder Structure.

File: routes.json (hunk 1 above)
Comment (Irutingabo) [reply routes.json]: Sorry, I removed it,
It had no use there, 
I was using it before.

File: webpack.config.js (hunk 2 above)
Comment (Irutingabo) [reply webpack.config.js]: Sure, It's Added!

**Review by placideirandora — APPROVED** [review]

**Pull request merged**